source/administration/mmctl-cli-tool.rst

```diff
@@ -1582,6 +1584,76 @@ mmctl config get
    --local                       allows communicating with the server through a unix socket
    --strict                      will only run commands if the mmctl version matches the server one
 
+mmctl config migrate
+^^^^^^^^^^^^^^^^^^^^
+
+**Description**
+
+Migrates a file-based configuration to (or from) a database-based configuration. Point the Mattermost server at the target configuration to start using it.
```

Comment:
@isacikgoz Does the server need to be restarted to apply the changes or can it be done without a restart? The config in database docs specify a restart is required https://docs.mattermost.com/administration/config-in-database.html#reload-systemd-files-and-restart-mattermost

Reply:
Actually this command only migrates the configuration data from one to another. To change the store type (file or a DB) the administrator needs to set `MM_CONFIG` environment variable and restart the server app.

Reply:
@wiersgallak Any additional updates needed for this PR from your perspective?

Reply:
@cwarnermm I think we should add the additional details about changing the store type as well.  If the admin is migrating, they likely want to have the changes be stored and actually use it ongoing.

Reply:
Thank you both -- this was a very good clarification to include in the product documentation.